www/src/components/utils/Toasts.js

```diff
@@ -63,3 +63,13 @@ export function SuccessToast({ children, onClose = () => {}, ...banner }) {
   )
 }
 
+export function InfoToast({ children, onClose = () => {}, ...banner }) {
```

Comment:
We may want to put this in the DS.

Reply:
The whole Toast component should be moved there, but since this was already a bigger change I only added this small wrapper method here.

Reply:
I'll do that in a separate PR. I have already added Toast to the DS